modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -3166,6 +3170,19 @@ private void _addOrUpdateResourcePermissions(
 					layoutPageTemplateEntry.getLayoutPageTemplateEntryId()));
 		}
 
+		List<ExpandoColumn> expandoColumns =
+			_expandoColumnLocalService.getExpandoColumns(-1, -1);
+
+		for (ExpandoColumn expandoColumn :
+			expandoColumns) {
+
+			stringUtilReplaceValues.put(
+				"EXPANDO_COLUMN_ID:" +
+				expandoColumn.getName(),
+				String.valueOf(
+					expandoColumn.getColumnId()));
+		}
+
```

Comment:
please move this behavior to a proper method responsible to handle expando columns